framework/src/kernels/ADVectorDiffusion.C

```diff
@@ -0,0 +1,34 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "ADVectorDiffusion.h"
+
+registerADMooseObject("MooseApp", ADVectorDiffusion);
+
+defineADValidParams(ADVectorDiffusion,
+                    ADVectorKernel,
+                    params.addClassDescription(
+                        "Same as `VectorDiffusion` in terms of physics/residual, but the Jacobian "
```

Comment:
General statement: I think our documentation for the `AD` objects shouldn't reference the non-AD objects and should be complete and stand on its own.  My reasoning?  Because AD will be "the way" eventually and we'll remove the non-AD objects.

Reply:
Update documentation...no longer refer to the non-AD object